tests/integration/features/ttl/ttl.feature

```diff
@@ -0,0 +1,23 @@
+Feature: room/ttp
```

Comment:
```suggestion
Feature: ttl/ttl
```
But I would suggest to move it into the chat/ folder
```suggestion
Feature: chat/ttl
```

A new folder means a new drone job, means more CI time is spent on cloning and setting up.
And since it's about chat messages only, chat/ sounds good enough?

Reply:
typo